x-pack/test_serverless/shared/config.base.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { format as formatUrl } from 'url';
+
+import { esTestConfig, kbnTestConfig, kibanaServerTestUser } from '@kbn/test';
+
+export default async () => {
+  const servers = {
+    kibana: kbnTestConfig.getUrlParts(),
+    elasticsearch: esTestConfig.getUrlParts(),
+  };
+
+  return {
+    servers,
```

Comment:
This morning I remembered that actually define timeouts in the config and since we know existing tests are failing mainly with timeout for serverless, what do we think about the current defaults:

https://github.com/elastic/kibana/blob/main/packages/kbn-test/src/functional_test_runner/lib/config/schema.ts#L112-L120

Probably no need to make decision for now

Reply:
Agree, it's good to open an easy door for setting these timeouts for serverless tests. In 58de8b85f1c4d85664031cf61cec5ee12a41ddbe, I've added a general timeout override to the serverless config (with copied numbers from current stateful tests), so we can easily adjust them where needed.